js_modules/dagster-ui/packages/ui-core/src/app/ExecutionSessionStorage.tsx

```diff
@@ -55,7 +55,7 @@ export interface IExecutionSession {
   base: SessionBase | null;
   mode: string | null;
   needsRefresh: boolean;
-  assetSelection: {assetKey: AssetKeyInput; opNames: string[]}[] | null;
+  assetSelection: {assetKey: AssetKeyInput; opNames?: string[]}[] | null;
```

Comment:
We've been passing around `opNames` as undefined from `DryRunRequest` but masking it on this line:
https://github.com/dagster-io/dagster/blob/22201160761c5b7741aabfbd0d859bf473f41d1c/js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadSetupRoot.tsx#L86 since we use `as any` and we also don't validate the type. 

So I'm marking this as optional to reflect the way its actually being used.